scripts/ci/ci_build_dockerhub.sh

```diff
@@ -67,6 +67,25 @@ echo
 # shellcheck source=scripts/ci/_script_init.sh
 . "$( dirname "${BASH_SOURCE[0]}" )/_script_init.sh"
 
+DOCKER_IMAGE_SPEC="${DOCKERHUB_USER}/${DOCKERHUB_REPO}:${DOCKER_TAG}"
+
+IMAGE_CREATED_SINCE=$(docker image ls "${DOCKER_IMAGE_SPEC}" --format '{{ .CreatedSince }}')
+
+if [[ ${IMAGE_CREATED_SINCE} =~ .*days.* || ${IMAGE_CREATED_SINCE} =~ .*weeks.* ||
+      ${IMAGE_CREATED_SINCE} =~ .*months.* || ${IMAGE_CREATED_SINCE} =~ .*years.* ||
+      ${IMAGE_CREATED_SINCE} =~ [0-9][0-9]\ hours ]]; then
```

Comment:
If we decide to not build image on every change then maybe we can run it every 24h?

Reply:
That's not possible on DockerHub. The build needs to be triggered by push to repo, there are no time triggers :(. We could use an external service for that and generate a special URL to trigger the builds but if it leaks we can be DDos-ed and it adds unnecessary third party. So I figured best thing is to check how fresh the image is.